js/data.js

```diff
@@ -453,6 +508,17 @@ var data = {
         dozent: ["Prof. Dr. Patrick Baudisch"],
         nameLV: "HCI Project Seminar Haptics, Personal Fabrication, and Virtual Reality"
     },
+    hciProjectSeminarPISME: {
+        kurz: "HCI Projekt<br />3D-Printing",
+        kennung: [ITSE, HCT, SAMT],
+        lehrform: ["Seminar", "Projekt"],
+        modul: ["IT Systems Engineering", "Vertiefungsgebiete"],
+        semester: ss15,
```

Comment:
Don't know. There are several courses in the file which do not have braces for the semester. I don't know if they were/are shown correctly. I added braces to the HCI project seminars.